doc/beanstalkd.ronn

```diff
@@ -96,6 +83,17 @@ and format of the `beanstalkd` protocol.
 * `-z` <bytes>:
   The maximum size in bytes of a job.
 
+* `-c`:
+  This flag has no effect. It is kept for historical compatibility only.
+
+  Perform online, incremental compaction of binlog files. Negates
```

Comment:
Do you think this line should be deleted?

Reply:
They have no meaning. Removed them.